src/test/java/uk/gov/hmcts/cmc/claimstore/rules/CountyCourtJudgmentRuleTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.cmc.claimstore.rules;
+
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.exceptions.ForbiddenActionException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class CountyCourtJudgmentRuleTest {
+
+    private final CountyCourtJudgmentRule countyCourtJudgmentRule = new CountyCourtJudgmentRule();
+
+    @Test
+    public void saveShouldFinishSuccessfullyForHappyPath() {
+        Claim claim = SampleClaim.builder().withResponseDeadline(LocalDate.now().minusMonths(2)).build();
+        countyCourtJudgmentRule.assertCanIssueCountyCourtJudgment(claim);
+    }
+
+    @Test(expected = ForbiddenActionException.class)
+    public void shouldThrowsForbiddenActionExceptionWhenClaimWasResponded() {
+        Claim respondedClaim = SampleClaim.builder().withRespondedAt(LocalDateTime.now().minusDays(2)).build();
+        countyCourtJudgmentRule.assertCanIssueCountyCourtJudgment(respondedClaim);
+    }
+
+    @Test(expected = ForbiddenActionException.class)
+    public void shouldThrowsForbiddenActionExceptionWhenUserCannotRequestCountyCourtJudgmentYet() {
+        Claim claim = SampleClaim.getWithResponseDeadline(LocalDate.now().plusDays(12));
+        countyCourtJudgmentRule.assertCanIssueCountyCourtJudgment(claim);
+    }
+
+    @Test(expected = ForbiddenActionException.class)
+    public void shouldThrowsForbiddenActionExceptionWhenCountyCourtJudgmentWasAlreadySubmitted() {
+        Claim claim = SampleClaim.getDefault();
```

Comment:
I might ask why default claim is not eligible for CCJ. I would use builder to set the reason of rejection explicitly. That will make test case more readable and make it independent of the sample claim defaults.

Reply:
done